README.md

```diff
@@ -10,8 +10,8 @@ whitelisting or SSL certificates manually.
 
 ### Examples
 
-For an examples of a simple application using Cloud SQL Socket Factory, please take a look at the 
-projects found 
+For examples of the SocketFactory being used in the context of an application, check out the sample 
```

Comment:
```suggestion
For examples of `SocketFactory` being used in the context of an application, check out the sample 
```

Reply:
I went more generic - I didn't want to accidentally encourage use of the `SocketFactory` class itself.